src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2111,7 +2111,11 @@ namespace UnitarySystems {
                     NoLoadHeatingAirFlowRateRatio =
                         min(this->m_NoLoadAirFlowRateRatio, state.dataUnitarySystems->designSpecMSHP[MSHPIndex].heatingVolFlowRatio[0]);
                 }
-                this->m_NoLoadAirFlowRateRatio = min(NoLoadCoolingAirFlowRateRatio, NoLoadHeatingAirFlowRateRatio);
+                if (state.dataUnitarySystems->designSpecMSHP[MSHPIndex].noLoadAirFlowRateRatio > 0.0) {
+                    this->m_NoLoadAirFlowRateRatio = state.dataUnitarySystems->designSpecMSHP[MSHPIndex].noLoadAirFlowRateRatio;
+                } else {
+                    this->m_NoLoadAirFlowRateRatio = min(NoLoadCoolingAirFlowRateRatio, NoLoadHeatingAirFlowRateRatio);
+                }
```

Comment:
added this logic such that when the `noLoadAirFlowRateRatio` greater than zero (this field is specified by user), then it uses this ratio to calculate the `Design Size No Load Supply Air Flow Rate`.